### PR TITLE
All activity

### PR DESCRIPTION
**Pull request by belucid**
This PR implements the all activity timeline API.

Supporting PR: https://github.com/open-company/open-company-lib/pull/17

To test:

### CODE REVIEW

File: src/oc/storage/resources/entry.clj [hunk 1]
```diff
@@ -150,6 +152,20 @@
...
+  and a direction, one of `:before` or `:after`, return the entries for the org with any interactions.
+  "
+  [conn org-uuid :- lib-schema/UniqueID order start :- lib-schema/ISO8601 direction]
+  {:pre [(db-common/conn? conn)
+          (#{:desc :asc} order)
+          (#{:before :after} direction)]}
+  (db-common/read-resources-and-relations conn table-name :org-uuid org-uuid
+                                          "created-at" order start direction config/default-limit
+                                          :interactions common/interaction-table-name :uuid :entry-uuid
+                                          ["uuid" "body" "reaction" "author" "created-at" "updated-at"]))
```
Comment (bago2k4) [inline src/oc/storage/resources/entry.clj]: Why you get the body but not headline here?

File: src/oc/storage/api/activity.clj [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+                        false))
+
+  ;; Responses
+  :handle-ok (fn [ctx] (let [user (:user ctx)
+                             user-id (:user-id user)
+                             org (:existing-org ctx)
+                             org-id (:uuid org)
+                             ;boards (board-res/list-boards-by-org conn org-id [:created-at :updated-at :authors :viewers :access])
+                             ;allowed-boards (map :uuid (filter #(access/access-level-for org % user) boards))
+                             ;board-uuids (map :uuid boards)
```
Comment (bago2k4) [inline src/oc/storage/api/activity.clj]: Why you left in these commented lines?

**Comment by bago2k4** [comment]
@belucid also tests are failing, looks like lein2 is not available anymore on Travis.

File: src/oc/storage/api/activity.clj (hunk 2 above)
Comment (belucid) [inline src/oc/storage/api/activity.clj]: @bago2k4 as a reminder, I need to filter out private boards from the user.

The reminder did its job!

File: src/oc/storage/resources/entry.clj (hunk 1 above)
Comment (belucid) [inline src/oc/storage/resources/entry.clj]: These are properties of an interaction (comment or reaction), not an entry, so they don't have headlines, only a body.

**Comment by bago2k4** [comment]
@bel since you need to still work on this it would be good to add explain the generate script in the README

**Pull request merged**